### PR TITLE
S3: reduce retires time for queries, increase retries count for backups

### PR DESCRIPTION
**Pull request by CheSema**
### Changelog category (leave one):
- Not for changelog

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
This PR was reverted

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

<details>
    <summary>Modify your CI run</summary>

**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step

#### Include tests (required builds will be added automatically):
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Unit tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with Analyzer
- [ ]  All with Azure
- [ ]  Add your option here

#### Exclude tests:
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with MSAN
- [ ]  All with UBSAN
- [ ]  All with Coverage
- [ ]  All with Aarch64
- [ ]  Add your option here

#### Extra options:
- [ ]  do not test (only style check)
- [ ]  disable merge-commit (no merge from master before tests)
- [ ]  disable CI cache (job reuse)

#### Only specified batches in multi-batch jobs:
- [ ]  1
- [ ]  2
- [ ]  3
- [ ]  4

</details>

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
*This is an automated comment for commit ff01ca80b4ae644adc396c0b94d3b54ac2b8c1e6 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/63857/ff01ca80b4ae644adc396c0b94d3b54ac2b8c1e6/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Review by vitlibar — APPROVED** [review]

File: src/Core/SettingsChangesHistory.h [hunk 1]
```diff
@@ -92,6 +92,8 @@ static std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges> sett
               {"cross_join_min_bytes_to_compress", 0, 1_GiB, "A new setting."},
               {"prefer_external_sort_block_bytes", 0, DEFAULT_BLOCK_SIZE * 256, "Prefer maximum block bytes for external sort, reduce the memory usage during merging."},
               {"input_format_force_null_for_omitted_fields", false, false, "Disable type-defaults for omitted fields when needed"},
+              {"output_format_pretty_preserve_border_for_multiline_string", 0, 1, "Applies better rendering for multiline strings."},
```
Comment (CheSema) [inline src/Core/SettingsChangesHistory.h:95]:
```suggestion
```

**Comment by CheSema** [comment]
test_checking_s3_blobs_paranoid/test.py::test_query_is_canceled_with_inf_retries -- Im fixing here https://github.com/ClickHouse/ClickHouse/issues/64451

**Comment by CheSema** [comment]
I do not understand that 
'''
Changed settings are not reflected in settings changes history (see changed_settings.txt)
New settings are not reflected in settings changes history (see new_settings.txt)
'''
The listed settings are not related.

**Comment by Algunenano** [comment]
> Upgrade check (debug) Changed settings are not reflected in settings changes history (see changed_settings.txt)

**Comment by Algunenano** [comment]
You added a new setting change in 24.5, not 24.6

**Comment by CheSema** [comment]
> You added a new setting change in 24.5, not 24.6

It is sad.
It became 24.6 while I was waiting CI.

**Pull request merged**